src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -134,33 +154,57 @@ private boolean isExceptionRecordCreated(UUID randomPoBox) {
             )
         );
 
-        exceptionRecordCase = caseDetailsList.isEmpty() ? null : caseDetailsList.get(0);
-        return exceptionRecordCase != null;
+        exceptionRecord = caseDetailsList.isEmpty() ? null : caseDetailsList.get(0);
+        return exceptionRecord != null;
     }
 
-    private void verifyExistingCaseIsUpdatedWithExceptionRecordCaseDetails() {
+    private Boolean isExceptionRecordAttachedToTheCase(CaseDetails caseDetails, int expectedScannedDocsSize) {
+
         CaseDetails updatedCase = caseRetriever.retrieve(
-            existingCase.getJurisdiction(),
-            String.valueOf(existingCase.getId())
+            caseDetails.getJurisdiction(),
+            String.valueOf(caseDetails.getId())
         );
 
         List<ScannedDocument> updatedScannedDocuments = getScannedDocumentsForSupplementaryEvidence(updatedCase);
 
-        List<ScannedDocument> scannedDocuments = getScannedDocumentsForExceptionRecord(exceptionRecordCase);
+        return !updatedScannedDocuments.isEmpty()
+            && updatedScannedDocuments.size() == expectedScannedDocsSize;
```

Comment:
`!updatedScannedDocuments.isEmpty()` is not needed

Reply:
ok, removed it now